fhir-server/src/test/java/org/linuxforhealth/fhir/server/util/FHIRRestHelperTest.java

```diff
@@ -2984,8 +2985,8 @@ public void testTransactionBundlePostWithResourceSchemeReference() throws Except
      */
     @Test
     public void testSearchWithPreviousPageResultsShiftWarning() throws Exception {
+        Random random = new Random();   
```

Comment:
we probably don't need random in this test, but i think its fine...the risk of a collission is EXTREMELY small